index.html

```diff
@@ -90,7 +93,7 @@ <h3>Project name goes...</h3>
               <li>Ruby on Rails</li>
               <li>+1</li>
             </ul>
-            <button type="button">See this project →</button>
+            <button type="button" data-modal-target>See this project →</button>
```

Comment:
- [ ] Kindly add button attribute to this button like `hover`, `active` etc. 👍 for accessibility reason

| Your design | Figma design |
| :---: | :---: |
| ![Capture 1](https://user-images.githubusercontent.com/80458199/233706629-d3c6933f-e7a7-4188-ac15-3d79eb189fa2.PNG) | ![Capture](https://user-images.githubusercontent.com/80458199/233706652-af40143f-3d6d-4b17-bdbd-ddd438b045be.PNG) |